.github/workflows/deploy.yml

```diff
@@ -50,20 +52,15 @@ jobs:
           npm run build
           touch build/.nojekyll
 
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v3
+      - name: Deploy to GitHub Pages
+        uses: JamesIves/github-pages-deploy-action@881db5376404c5c8d621010bcbec0310b58d5e29
```

Comment:
What is this?

Reply:
JamesIves is the sort of "standard" old way to deploy github pages. We have the commit sha there instead of a version as we allowlist non-github actions.